wikipediapreview-wordpress.php

```diff
@@ -1,6 +1,13 @@
 <?php
 /**
  * Plugin Name: Wikipedia Preview
- * Description: Add Wikipedia Preview to your articles
- * Version: 0.1.0
+ * Plugin URI: https://github.com/wikimedia/wikipedia-preview
+ * Description: Wikipedia Preview allows you to show a popup card with a short summary from Wikipedia when a reader clicks or hovers over a link
+ * Version: 1.1.1
+ * Requires at least: 5.6
+ * Requires PHP: 8.0.2
```

Comment:
i think we don't really need to specific the php version here, we don't use any special syntax in php, therefore the older version like php 7 (or even older) should still support, given that the site may in the older version.

Reply:
Actually from my Local app dashboard, the php version for my local site is listed as 7.3.5 (and Wikipedia Preview is working of course). What's yours?

Reply:
same, and I changed to 8.0.0.

Reply:
I think we should lower the PHP requirement to 7.3.5 at least then to make it more accessible, makes sense?

Reply:
our plugin mainly focus in JavaScript, nothing fancy php so the php version shouldn't be a blocker

what if we keep "Requires PHP" empty?